### PR TITLE
FIT: new feature("--file-for") for digit2raw

### PR DESCRIPTION
**Pull request by afurs**

### CODE REVIEW

**Comment by afurs** [comment]
@shahor02 , @davidrohr as you requested, new "--file-for" feature is prepared for FIT's digit2raw. FLP name could be defined via --flp-name(by default "alio2-cr1-flp180"). Because 2 new FLPs are not installed yet for FIT(3 in total, 1 for each subdetector) and we don't know new flp names, I putted "-fv0"/"-fdd" suffix in default flp name. And "--file-per-link" is removed

**Review by shahor02 — DISMISSED** [review]
Thanks!

**Review by davidrohr — CHANGES_REQUESTED** [review]
Looks good, but you will also need to change the option in O2/prodtests/full-system-test/dpl-workflow.sh, otherwise the FullCI will fail

**Comment by afurs** [comment]
@davidrohr but what exactly should I change in `O2/prodtests/full-system-test/dpl-workflow.sh`? I didn't find any sign of `digit2raw`. Didi you mean `prodtests/full-system-test/create_full_system_pipeline.py` ?

**Comment by davidrohr** [comment]
@afurs : Oh, sorry, I meant `prodtests/full_system_test.sh?

**Comment by afurs** [comment]
@davidrohr , ok, it's done for `full_system_test.sh` but I also changed options in 4f52165 Should I cancel it?

**Comment by davidrohr** [comment]
> @davidrohr , ok, it's done for `full_system_test.sh` but I also changed options in [4f52165](https://github.com/AliceO2Group/AliceO2/commit/4f52165086203130a3d4a89fc8feeacf8e81ae5d) Should I cancel it?

no, everything is fine now (the other file is currently not used and not checked in the CI, so would not have broken the CI)

File: Detectors/FIT/FT0/simulation/src/digit2raw.cxx [hunk 1]
```diff
@@ -48,7 +48,8 @@ int main(int argc, char** argv)
     add_option("verbosity,v", bpo::value<int>()->default_value(0), "verbosity level");
     //    add_option("input-file,i", bpo::value<std::string>()->default_value(o2::base::NameConf::getDigitsFileName(o2::detectors::DetID::FT0)),"input FT0 digits file"); // why not used?
     add_option("input-file,i", bpo::value<std::string>()->default_value("ft0digits.root"), "input FT0 digits file");
-    add_option("file-per-link,l", bpo::value<bool>()->default_value(false)->implicit_value(true), "create output file per CRU (default: per layer)");
+    add_option("flp-name", bpo::value<std::string>()->default_value("alio2-cr1-flp180"), "single file per: all,flp,cru,link");
```
Comment (davidrohr) [inline Detectors/FIT/FT0/simulation/src/digit2raw.cxx:51]: Actually, the comment for this option seems to be wrong, I guess copy&paste error.

Comment (afurs) [reply Detectors/FIT/FT0/simulation/src/digit2raw.cxx:51]: yes, sorry, i can change it in next PR(in few hours)

Comment (afurs) [reply Detectors/FIT/FT0/simulation/src/digit2raw.cxx:51]: oh, I meant after merging this one :) It will be not in few hours

**Comment by afurs** [comment]
@davidrohr @shahor02 there are conflict in `prodtests/full_system_test.sh`

**Comment by davidrohr** [comment]
@afurs : ok, never mind, that happens now since multiple detectors have to edit the same file. If the fullCI succeeds I'll resolve the conflict manually and merge it. Otherwise you'll have to resolve and push a new commit.

**Comment by davidrohr** [comment]
CI had passed, just merged the conflict with prior change in TPC part of FST, merging

**Pull request merged**